__tests__/app.test.js

```diff
@@ -58,3 +58,30 @@ afterAll(() => {
         });
       });
 
+      describe('GET /api/articles/:article_id', () => {
+        test('responds with an article object', () => {
+          return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+              const article = response.body.article;
+              expect(article).toHaveProperty('author');
+              expect(article).toHaveProperty('title');
+              expect(article).toHaveProperty('article_id');
+              expect(article).toHaveProperty('body');
+              expect(article).toHaveProperty('topic');
+              expect(article).toHaveProperty('created_at');
+              expect(article).toHaveProperty('votes');
+              expect(article).toHaveProperty('article_img_url');
+            });
+        });
```

Comment:
There's a missing error handling test to do here for an invalid ID of an article (i.e. /api/articles/notAnID), along with the 400 error code